Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerStats.h

```diff
@@ -20,11 +22,22 @@ namespace AzNetworking
 
 namespace Multiplayer
 {
+    enum MultiplayerGroupIds
+    {
+        MultiplayerGroup_Networking
+    };
+
+    enum MultiplayerStatIds
+    {
+        MultiplayerStat_EntityCount = 1000,
```

Comment:
not clear to me why this is set to 1000?

Reply:
No reason, it can be any number but I like to start somewhere high enough to avoid collisions with other defined values of stat ids. For example, if we were to add them in o3de-multiplayersample